test/unit/controller/user-controller.ts

```diff
@@ -1193,7 +1193,7 @@ describe('UserController', (): void => {
           'TransactionReportResponse',
           res.body,
           false,
-          true,
+          false,
```

Comment:
If I remember correctly, we now allow more data to be returned than just the spec? Why would we suddenly allow this?

Reply:
Same goes as for the previous remark `TransactionReportResponse` - `data.entries.product` is of type `BaseProductResponse` but also has a `revision` property.

Reply:
Same as above. Please add a comment that elaborates why this test is different than all the others (and include the Github issue reference).